### DISCUSSION TITLE
Custom prompts

### DISCUSSION THREAD

**Discussion by aarnphm** [yetone/avante.nvim#468] [Show and tell]
With the introduction of https://github.com/yetone/avante.nvim/pull/466, custom prompts are now supported for each mode.

This serves as a thread for everyone to share their prompts for their specific use case.

Please do keep in mind to follow guidelines, no NSFW contents, be nice and also, and no VSCode or CURSOR lmao

**Comment by aarnphm** [comment]
example of `typescript.planning.avanterules`:

```jinja
{% extends "planning.avanterules" %}
{% block user_prompt %}
Your tasks is to only EDIT TYPESCRIPT FILE. If it is not a TYPESCRIPT FILE SAY "MY ONLY TASKS IS TO WORK ON TYPESCRIPT, AS I'M ONLY GOOD AT TYPESCRIPT"
{% endblock %}
```

**Comment by luxus** [comment]
does anyone have a prompt for svelte?
